.travis.yml

```diff
@@ -20,3 +20,15 @@ before_script:
 
 script:
   - "$SCRIPT"
+
+matrix:
+  include:
+    - rust: stable
+      before_script:
+        - rustup target add i686-pc-windows-gnu
+      script:
+        - cd ./wee_alloc
+        - cargo check --release --features "size_classes"
+        - cargo check --release
```

Comment:
I believe these are wrong - `size_classes` is enabled by default, so one of the lines had to remain `cargo check --release` but another is supposed to be `cargo check --release --no-default-features`.